sections/dashboard/EligibilityBox/EligibilityBox.tsx

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useState } from 'react';
+import styled from 'styled-components';
+
+import Button from 'components/Button';
+import Input from 'components/Input/Input';
+import Text from 'components/Text';
+
+import EligibilityMap from './EligibilityMap';
```

Comment:
As it's a constant I think we should uppercase as `ELIGIBILITY_MAP`

Reply:
Changed.